Lib/multiprocessing/process.py

```diff
@@ -110,6 +110,9 @@ def start(self):
         _cleanup()
         self._popen = self._Popen(self)
         self._sentinel = self._popen.sentinel
+        # Avoid a refcycle if the target function holds an indirect
+        # reference to the process object
+        del self._target, self._args, self._kwargs
```

Comment:
I dislike magic attributes which disappear. Would it be possible to assign them to None instead? Please add a reference to bpo-30775 in the comment.